src/main/main.c

```diff
@@ -16,6 +16,6 @@ int	main(int argc, char **argv)
 {
 	(void)argc;
 	(void)argv;
-	printer_print("~Minishell~");
+	printf("~Minishell~\n\n");
```

Comment:
same here...
it is better to not push changes like this since they are redundent and make it harder to do CR

Reply:
Since i did my tests directly from main It became a mess, I didn't take that into account when going back